File: Hello/app/src/main/java/com/example/hello/MainActivity.java
package com.example.hello;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    //MainActivity 가 최초 실행될 때 실행된다.
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //Activity의 UI를 R.layout.activity_main으로 지정한다.
        setContentView(R.layout.activity_main);

        //"프로그래밍을 시작해보자!" 메세지를 잠시 보여준다.
        Toast.makeText(getApplicationContext(),"프로그래밍을 시작해보자!",Toast.LENGTH_LONG).show();
    }
}

File: Project/KotlinSample/app/src/main/java/com/example/kotlinsample/MainActivity.kt
package com.example.kotlinsample

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        button1.setOnClickListener{
//            startActivity(Intent(this@MainActivity, BmiJavaActivity::class.java))
//        }
//
//        button2.setOnClickListener{
//            startActivity(Intent(this@MainActivity,BmiKotlinActivity::class.java))
//        }

        button3.setOnClickListener {
            startActivity(Intent(this@MainActivity,VariableJavaActivity::class.java))
        }
        button4.setOnClickListener {
            startActivity(Intent(this@MainActivity,VariableKotlinActivity::class.java))
        }
        button5.setOnClickListener {
            startActivity(Intent(this@MainActivity,ControlJavaActivity::class.java))
        }
        button6.setOnClickListener {
            startActivity(Intent(this@MainActivity,ControlKotlinActivity::class.java))
        }
    }
}
